src/System.Linq/src/System/Linq/Skip.cs

```diff
@@ -121,5 +121,33 @@ private static IEnumerable<TSource> SkipWhileIterator<TSource>(IEnumerable<TSour
                 }
             }
         }
+
+#if netcoreapp11
+        public static IEnumerable<TSource> SkipLast<TSource>(this IEnumerable<TSource> source, int count)
+        {
+            if (source == null)
+            {
+                throw Error.ArgumentNull(nameof(source));
+            }
+
+            return SkipLastIterator(source, Math.Max(count, 0));
+        }
+
+        private static IEnumerable<TSource> SkipLastIterator<TSource>(IEnumerable<TSource> source, int count)
+        {
+            var queue = new Queue<TSource>();
+
+            foreach (TSource item in source)
+            {
+                if (queue.Count == count)
+                {
+                    yield return item;
+                    queue.Dequeue();
```

Comment:
@JonHanna CI is failing... I mentioned in https://github.com/dotnet/corefx/issues/13842#issuecomment-264613570 that I was having trouble to get the tests to run (or even compile) locally, since all of the netcoreapp1.1 tests were being excluded. Luckily however they seem to be running in CI.